command/resource/read/read_test.go

```diff
@@ -7,6 +7,10 @@ import (
 
 	"github.com/mitchellh/cli"
 	"github.com/stretchr/testify/require"
+
+	"github.com/hashicorp/consul/agent"
+	"github.com/hashicorp/consul/command/resource/apply"
+	"github.com/hashicorp/consul/testrpc"
 )
 
 func TestResourceReadInvalidArgs(t *testing.T) {
```

Comment:
We should add a test case where the resource type is in an invalid format
example: `args: []string{"test"}`

Reply:
Yup, as discussed above, we throw error if it's not the format of `a.b.c`, and added one test case above